r2dbc-spi-test/src/main/java/io/r2dbc/spi/test/MockRowMetadata.java

```diff
@@ -50,6 +53,12 @@ public ColumnMetadata getColumnMetadata(Object identifier) {
         return this.columnMetadatas;
     }
 
+    @Override
+    public SortedSet<String> getColumnNames() {
+        return columnMetadatas.stream().map(ColumnMetadata::getName).collect(Collectors.toCollection(
```

Comment:
Due to performance problems with the `Stream` APIs, we've gotten away from using them, even in situations like this.  Can you please re-implement this equivalently with a `for` loop?

Reply:
As recommended, replaced `Stream` with `forEach`

Reply:
That should be rather a loop. forEach creates a capturing lambda until it gets optimized away by the JIT compiler.

Reply:
Thanks, @mp911de, I  changed to loop.